### PR TITLE
Move flags and polishing

### PR DESCRIPTION
**Pull request by ghost**
Tested and works <3

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -591,6 +591,7 @@ let BattleMovedex = {
 			},
 			onStart: function (target, source) {
 				this.add('-fieldstart', 'move: Trivia Room', '[of] ' + source);
+				this.add('-message', `Bimp is levitating due to its big trivia brain!`); // Bimp is the only one levitating in trivia room
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: source.side.name

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -600,6 +601,7 @@ let BattleMovedex = {
 			onResidualOrder: 23,
 			onEnd: function () {
 				this.add('-fieldend', 'move: Trivia Room');
+				this.add('-message', `Bimp is no longer levitating.`);
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: source.side.name

File: mods/seasonal/moves.js [hunk 3]
```diff
@@ -3209,10 +3211,10 @@ let BattleMovedex = {
 			// Generate new set
 			const generator = new RandomStaffBrosTeams('gen7randombattle', this.prng);
 			let set = generator.randomSet(claim);
+			// Suppress Ability now to prevent starting new abilities when transforming
+			source.addVolatile('gastroacid', source);
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:3215]: This is undone when he switches out though, the question becomes will the ability be correct then too?

Comment (ghost) [reply mods/seasonal/moves.js:3215]: *discord conversation*

**Pull request merged**